### PR TITLE
Addition of HybridDataManager SDK

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please set this to `0.9.0-preview`

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please use [this](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L11) as an example

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: For consistency please name this as `Microsoft.Azure.Management.HybridData`

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please add the `PackageReleaseNotes`

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please remove this `PropertyGroup`

File: src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj]: Please use [this](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L18) as a sample

File: src/SDKs/HybridDataManager/Management.HybridData/generate.ps1
Comment (dsgouda) [inline src/SDKs/HybridDataManager/Management.HybridData/generate.ps1]: Is the Rest spec PR merged already?

File: README.md
Comment (dsgouda) [inline README.md]: These changes are unexpected. Please revert this file to the remote master

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/HybridDataManager/Management.HybridData/generate.ps1
Comment (t-prta) [reply src/SDKs/HybridDataManager/Management.HybridData/generate.ps1]: No, we are waiting for a linter suppression approval for that PR

File: README.md
Comment (t-prta) [reply README.md]: Pulling the master/readme.md and committing it.

Comment (t-prta) [reply README.md]: The file is an exact copy of azure/azure-sdk-for-net/master readme.md. I am not sure why the diff is showing up this way.

Comment (dsgouda) [reply README.md]: How are you reverting (git checkout ?)

**Comment by dsgouda** [comment]
Please link the rest spec pr here

File: README.md
Comment (t-prta) [reply README.md]: git checkout upstream/master -- README.md
I ran this command, and it doesn't modify the file.

Comment (dsgouda) [reply README.md]: Can you post what your upstream is here

Comment (t-prta) [reply README.md]: Upstream is the Azure/azure-sdk-for-net repo

**Comment by dsgouda** [comment]
Closing since a new PR is opened

**Pull request closed without merging**
